DNN Platform/Library/Common/Globals.cs

```diff
@@ -3514,19 +3602,18 @@ public static string SerializeHashTableBase64(Hashtable Source)
             return strString;
         }
 
-        /// -----------------------------------------------------------------------------
         /// <summary>
         /// SerializeHashTableXml serializes a Hashtable using Xml Serialization.
         /// </summary>
         /// <remarks>
         /// This is the preferred method of serialization under Medium Trust.
         /// </remarks>
-        /// <param name="Source">The Hashtable to serialize.</param>
+        /// <param name="source">The Hashtable to serialize.</param>
         /// <returns>The serialized String.</returns>
-        /// -----------------------------------------------------------------------------
-        public static string SerializeHashTableXml(Hashtable Source)
+        [Obsolete("Deprecated in v9.8.1, this method was never meant to be public and only works for 'profile' root namespace, scheduled removal in v10.")]

```

Comment:
Why is this being removed in v10 instead of v11?

Reply:
Addressed in latest commit, thanks.